src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -217,6 +218,41 @@ struct ParticlePositionVelocityCompute : ParticlePositionVelocity<Dim>,
 };
 /// @}
 
+/*!
+ * \brief The position of the scalar charge evolved by the worldtube singleton.
+ */
+template <size_t Dim>
+struct EvolvedPosition : db::SimpleTag {
```

Comment:
It's a bit confusing that you have duplicate tags for the position/velocity of the scalar charge. The notes on the compute tags are helpful, but I'm wondering if you could reuse the `ParticlePositionVelocity` simple tag on both the elements and the worldtube singleton, but then only have different compute tags? That way it's clear the position/velocity are the same thing, but they are just computed differently on different components (and maybe explain how they are computed differently?)

Reply:
Yes, I agree this is not optimal. One of the things to consider is that the `EvolvedPosition` and `EvolvedVelocity` tags are tensors of size 1 DataVectors as the time stepper can not advance doubles afaik. I try and hide this conversion from DataVectors to doubles in the compute tags but if I were to use this tag on the element components as well they would have to do this manually.

Reply:
@wthrowe Correct me if I'm wrong, but I thought the time stepper and history were able to handle `double`s?

Reply:
Hmm ok. Seems like it's not trivial to combine them then. In that case, please add a note to the `EvolvedPosition` and `EvolvedVelocity` tag docs that these tags are intended to be used on the worldtube singleton, not the elements.